File: sdkwork-app-sdk-csharp/Models/AdvertSettingsVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class AdvertSettingsVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public bool? Enabled { get; set; }
        public bool? Personalized { get; set; }
        public int? MaxDailyAds { get; set; }
        public string? AdBlockLevel { get; set; }
        public bool? RewardVideoEnabled { get; set; }
        public bool? InterstitialEnabled { get; set; }
        public bool? BannerEnabled { get; set; }
        public string? LastUpdated { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/advert_settings_vo.go
package types

// 广告设置VO
type AdvertSettingsVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Enabled bool `json:"enabled"`
	Personalized bool `json:"personalized"`
	MaxDailyAds int `json:"maxDailyAds"`
	AdBlockLevel string `json:"adBlockLevel"`
	RewardVideoEnabled bool `json:"rewardVideoEnabled"`
	InterstitialEnabled bool `json:"interstitialEnabled"`
	BannerEnabled bool `json:"bannerEnabled"`
	LastUpdated string `json:"lastUpdated"`
}
